lib/puppet/provider/oneview_logical_interconnect/c7000.rb

```diff
@@ -45,6 +49,15 @@ def destroy
     raise('This resource relies on others to be destroyed.')
   end
 
+  def bulk_inconsistency_validate
+    inconsistency_validation = @resource_type.new(@client)
+    inconsistency_validation.data['uri'] = 'dummy'
```

Comment:
change dummy to "@data['logical_interconnect_uris'].first"

Reply:
its added now